jira/server/scio_search/src/main/java/ScioSearchConfigRestPlugin/impl/IssueSecuritySchemeResponse.java

```diff
@@ -0,0 +1,22 @@
+package ScioSearchConfigRestPlugin.impl;
+
+import org.codehaus.jackson.annotate.JsonAutoDetect;
+
+import java.util.List;
+
+@JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
+public class IssueSecuritySchemeResponse {
+    public String id;
+    public String name;
+    public String description;
+    public String defaultSecurityLevelId;
+    public List<IssueSecurityLevel> levels;
+
+    @JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
+    public static class IssueSecurityLevel {
```

Comment:
should we keep this class outside of the response class?

Reply:
This class is related to IssueSecurity Scheme Response and not being used anywhere else. So I kept it inside. wdyt?